devito/ir/support/utils.py

```diff
@@ -158,3 +159,44 @@ def detect_io(exprs, relax=False):
             writes.append(f)
 
     return filter_sorted(reads), filter_sorted(writes)
+
+
+def expr_lowering(expressions, **kwargs):
```

Comment:
I don't think this is where I suggested this should go and I don't think this is how it should be named